### DISCUSSION TITLE
artisan and HB Standard Roaster not connecting since update on macOS 12.1

### DISCUSSION THREAD

**Discussion by InaBina1411** [artisan-roaster-scope/artisan#776] [Q&A] [answered]
I´m an artisan newbie and quite incapable in IT-things😔.... After having updated my MacBook (macOS 12.1.) I'm not able to connect my HB Roaster with artisan software anymore. I installed the latest artisan version (artisan-mac 2.4.6.) and driver (CP210xVCP). I'm really desperate, can somebody help????

**Comment by CarefreeBuzzBuzz** [comment, marked as answer]
Did you approve Artisan and the driver in security settings if needed. 

___________________________
Michael Herbert, CarefreeBuzzBuzz
